FlowCryptCommon/Extensions/StringExtensions.swift

```diff
@@ -45,6 +45,20 @@ public extension String {
             }
         }
     }
+    
+    var fileExtension: String? {
+        guard let fileExtension = self.split(separator: ".").last else {
+            return nil
+        }
+        return String(fileExtension)
+    }
```

Comment:
seems like a bug? file named just `pgp` will be seen as having extension pgp, which it doesn't.

Reply:
removed as unused